core/src/test/java/io/confluent/kafka/schemaregistry/ClusterTestHarness.java

```diff
@@ -173,12 +174,19 @@ public void setUp() throws Exception {
     }
 
     if (setupRestApp) {
-      restApp = new RestApp(choosePort(), srZkConnect, zkConnect, KAFKASTORE_TOPIC,
-                            compatibilityType, true);
+      schemaRegistryPort = choosePort();
+      Properties schemaRegistryProps = getSchemaRegistryProperties();
+      schemaRegistryProps.put(SchemaRegistryConfig.LISTENERS_CONFIG,getSchemaRegistryProtocol()+
```

Comment:
I am surprised checkstyle isn't catching this, but things are really compressed here w/ no spaces